yiyuzhang0604/SysopsMonolithicSpringBoot
# Add Okta for user authentication

To test get api: 
http://localhost:8088/okta/secured, when try to send, will get an error 

Now go to: https://dev-60403104.okta.com/oauth2/default/v1/token, and try to send a post request 
- should include x-www-form-urlencoded in the body tab: add key: grant_type, value : client_credentials, and another key: scope: defaultscope
- click authorization tab: select basic bearer token 
- after send the post request, copy the access token from response 
- add the token to the get request, now it should be successful.
